tests/vrt/anon_urls.txt

```diff
@@ -0,0 +1,20 @@
+https://reliefweb.int/
+https://reliefweb.int/updates
+https://reliefweb.int/jobs
+https://reliefweb.int/training
+https://reliefweb.int/disasters
+https://reliefweb.int/countries
+https://reliefweb.int/organizations
+https://reliefweb.int/topics
+https://reliefweb.int/blog
+https://reliefweb.int/report/occupied-palestinian-territory/delayed-denied-and-deprived-collective-punishment-palestinian-patients-gaza-context-israels-15-year-blockade
+https://reliefweb.int/map/myanmar/myanmar-place-origin-townships-displaced-population-camp-profiling-round-12-kachin-and-shan-state-north-may-2022
+https://reliefweb.int/job/3856494/ukraine-crisis-response-medical-coordinator-mf
+https://reliefweb.int/training/3850078/organised-crime-and-security-policy-understanding-threat-design-better-responses
+https://reliefweb.int/blogpost/onwards-and-upwards-reliefweb-turns-25-record-numbers
+https://reliefweb.int/country/afg
+https://reliefweb.int//disaster/fl-2021-000084-tcd
```

Comment:
```suggestion
https://reliefweb.int/disaster/fl-2021-000084-tcd
```

Reply:
Fixed (though it's re-directs to the right url automatically, which is how we hadn't picked it up before)